package.json

```diff
@@ -34,33 +33,58 @@
   },
   "devDependencies": {
     "@alrra/travis-scripts": "^3.0.1",
+    "del": "^3.0.0",
     "eslint": "^4.4.1",
+    "globby": "^7.1.1",
+    "gulp": "^4.0.0",
+    "gulp-brotli": "^1.2.1",
+    "gulp-changed": "^3.2.0",
+    "gulp-clean-css": "^3.9.2",
+    "gulp-cli": "^2.0.0",
+    "gulp-concat": "^2.6.1",
+    "gulp-debug": "^3.2.0",
+    "gulp-filter": "^5.1.0",
+    "gulp-flatten": "^0.4.0",
+    "gulp-htmlmin": "^4.0.0",
+    "gulp-if": "^2.0.2",
+    "gulp-imagemin": "^4.1.0",
+    "gulp-load-plugins": "^1.5.0",
+    "gulp-rev": "^8.1.1",
+    "gulp-rev-delete-original": "^0.2.3",
+    "gulp-rev-replace": "^0.4.3",
+    "gulp-server-livereload": "^1.9.2",
+    "gulp-uglify": "^3.0.0",
+    "gulp-useref": "^3.1.4",
+    "gulp-zopfli": "^1.0.0",
     "markdownlint": "^0.6.4",
     "markdownlint-cli": "^0.6.0",
     "mktemp": "^0.4.0",
     "npm-run-all": "^4.1.2",
+    "pump": "^2.0.0",
     "shelljs": "^0.7.7",
-    "stylelint": "^8.4.0"
+    "stylelint": "^8.4.0",
+    "useref": "^1.4.0"
   },
   "hexo": {
     "version": "3.4.3"
   },
   "private": true,
   "scripts": {
-    "build": "npm run hexo:clean && npm run update-content && node helpers/updater.js src/hexo/source && hexo generate",
+    "build": "npm run hexo:clean && npm run update-content && node helpers/updater.js src/content && gulp",
     "hexo:clean": "hexo clean",
-    "hexo:generate": "hexo generate",
-    "hexo:server": "hexo server",
+    "server": "npm run build -- watch",
+    "gulp": "gulp",
     "lint": "npm-run-all lint:*",
     "lint:browser": "eslint -c .eslintrc --env browser 'src/hexo/themes/sonarwhal/source/**/*.js'",
     "lint:js": "npm run lint:node && npm run lint:browser",
     "lint:md": "markdownlint README.md ./src/hexo/source",
     "lint:node": "eslint -c .eslintrc --env node '**/*.js' --ignore-pattern 'src/hexo/themes/sonarwhal/source/**/*.js' --ignore-pattern 'dist/**/*.js'",
     "lint:styles": "stylelint src/hexo/themes/sonarwhal/source/**/*.css --config .stylelintrc",
     "start": "node src/server/index.js",
+    "start:prod": "set NODE_ENV=production&& npm run start",
```

Comment:
> set NODE_ENV=production\<space\>&&

Reply:
For future reference (and because we can't put comments on `package.json`) here goes the link: https://stackoverflow.com/questions/25112510/how-to-set-environment-variables-from-within-package-json-node-js#comment58812038_27090755